### PR TITLE
Organise handlers into separate files

### PR DESCRIPTION
**Pull request by markwinter**
**What this PR does / why we need it**:

Currently handlers are split between `model_server.py` and `handlers/http.py`.

This PR moves the handlers to separate files under the `handlers` directory. I think this makes things a bit more organised

### CODE REVIEW

File: python/kserve/kserve/handlers/explain.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import json
```
Comment (yuzisun) [inline python/kserve/kserve/handlers/explain.py:15]: Can you help add the license on top?

File: python/kserve/kserve/handlers/base.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Copyright 2021 The KServe Authors.
```
Comment (yuzisun) [inline python/kserve/kserve/handlers/base.py]: 2021 -> 2022

File: python/kserve/kserve/handlers/liveness.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+from kserve.handlers.base import BaseHandler
+
+
+class LivenessHandler(BaseHandler):  # pylint:disable=too-few-public-methods
```
Comment (yuzisun) [inline python/kserve/kserve/handlers/liveness.py]: this can be merged to the `health.py`?

File: python/kserve/kserve/handlers/list.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from kserve.handlers.base import BaseHandler
+from kserve.model_repository import ModelRepository
+
+
+class ListHandler(BaseHandler):
```
Comment (yuzisun) [inline python/kserve/kserve/handlers/list.py]: I think `ListHandler` and `LoadHandler` can be merged into `model_management.py`

**Comment by markwinter** [comment]
@yuzisun Thanks for reviewing. I've applied your suggestions.

**Comment by yuzisun** [comment]
Thanks @markwinter for the nice refactoring!

/lgtm
/approve

**Pull request merged**